Tests/Expected/XCAssets/swift3-context-all-customname.swift

```diff
@@ -2,11 +2,11 @@
 
 #if os(OSX)
   import AppKit.NSImage
-  typealias Color = NSColor
+  typealias AssetColorTypeAlias = NSColor
   typealias Image = NSImage
 #elseif os(iOS) || os(tvOS) || os(watchOS)
   import UIKit.UIImage
-  typealias Color = UIColor
+  typealias AssetColorTypeAlias = UIColor
```

Comment:
We should use the occasion of this test case to also customise the color & image typealias names in addition to the enumName being customised to `XCTColorAsset` (same for all `*-customname.swift` test & expected of course). This way we're sure we didn't forget to replace some hardcoded `Color` identifier with `{{colorAlias}}` at some point.